cli/cmd/simulation/clustercomplete.hpp

```diff
@@ -0,0 +1,202 @@
+//
+// Created by Willem Lambooy on 07/03/2024.
```

Comment:
In ``fiction``, we use dots, e.g., 07.03.24.

Reply:
No Dutch pride allowed ;_;